web/pdf_find_controller.js

```diff
@@ -251,6 +251,10 @@ class PDFFindController {
           // Store the pageContent as a string.
           this.pageContents[i] = strBuf.join('');
           extractTextCapability.resolve(i);
+        }, (reason) => {
```

Comment:
Not exact science. Mostly I'm doing it by observation of then-handler: 1) if there is a logic that will cause exception or return a promise -- `.catch()` will be used; 2) however if logic have some simple finalization code thus no reason for exception, I'm trying to fit into then-catch-handler, it will avoid to perform additional micro task spawning.